assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/data/types.ts

```diff
@@ -0,0 +1,48 @@
+import { Reducer, ReducerState } from 'react';
+
+import { EntityId } from '@appServices/apollo/types';
+import { Ticket } from '@edtrServices/apollo';
+import { BaseProps, TpcPriceModifier } from '../types';
+
+export interface DataState {
+	ticket: Partial<Ticket>;
+	prices: Array<TpcPriceModifier>;
+	deletedPrices: Array<EntityId>;
+}
+
+export type DataActionType =
+	| 'UPDATE_TICKET_PRICE'
+	| 'TOGGLE_REVERSE_CALCULATE'
+	| 'ADD_PRICE'
+	| 'UPDATE_PRICE'
+	| 'DELETE_PRICE'
+	| 'ADD_PRICE_TO_DELETED'
```

Comment:
I'm assuming:

- DELETE_PRICE is for simply removing newly added modifiers

- ADD_PRICE_TO_DELETED is for scheduling existing prices for deletion

Reply:
- `DELETE_PRICE` removes a price modifier
- `ADD_PRICE_TO_DELETED` adds the deleted price ID to the bucket which we can use in `onSubmit`